webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -230,8 +240,9 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     return this.authService.has('can_view_last_visited_date');
   }
 
-  private formatContacts(contacts) {
-    return contacts.map(updatedContact => {
+  private formatContacts(contacts) { //NOSONAR
+    const trackPerformance = this.performanceService.track('contact-list:query:format_contacts');
+    const formattedContacts = contacts.map(updatedContact => { //NOSONAR
```

Comment:
Not the scope of this work, I created[ a ticket for that ](https://github.com/medic/cht-core/issues/8803)